index.html

```diff
@@ -9,37 +9,213 @@
-        <section class="intro-header">
+    <header>
+        <nav class="nav-bar">
+            <ul>
+                <li id="name"> <a href="#">Michael Oladele</a></li>
+                <li class="hamburger-icon"><a href="#"><i class="fas fa-bars"></i></a></li>
+                <li class="right-item"> <a href="#">Portfolio</a></li>
+                <li class="right-item"> <a href="#">About</a></li>
+                <li class="right-item"> <a href="#">Contact</a></li>
+                <li class="right-item"> <a href="#"><i class="fas fa-envelope-square"></i></a></li>
+            </ul>
+        </nav>
+       </header>
+ <section class="overall-container">
+    <section class="main-container">
+        <div class="intro-header">
             <h1>Hey There.
                 I’m Michael</h1>
-        </section>
-        <section class="intro-title">
-            <h3>I’am a Software Developer</h3>
-        </section>
-        <section class="intro-detail">
+        </div>
+       <div>
+        <div class="intro-detail">
+            <h1 class="intro-title">I’am a Software Developer</h1>
             <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
                 don’t hestiate to contact me.</p>
-        </section>
-        <section class="social">
+        </div>
+       </div>
+        <div class="social">
             <ul class="social-ul">
                 <li><a href="#"><i class="fab fa-github"></i></a></li>
                 <li><a href="#"><i class="fab fa-linkedin-in"></i></a></li>
                 <li><a href="#"><i class="fab fa-angellist"></i></a></li>
                 <li><a href="#"><i class="fab fa-twitter"></i></a></li>
                 <li><a href="#"><i class="fab fa-medium-m"></i></a></li>
             </ul>
-        </section>
-    </main>
+        </div>
+    </section>
+    <!-- My recent work section -->
+    <section class="my-projects-section">
+        <div id="projects-title">
+        <h1>My Recent Works</h1>
+        </div>
+        <div class="work-hero-img">
+            <img src="./images/works-hero-img.svg" alt="">
+        </div>
+        <div class="multi-post">
+            <h1 class="post-title">Multi-Post Stories</h1>
+            <div class="post">
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            </div>
+            <div class="tech-lang">
+                <input type="button" value="css" class="tech-item item-one"/>
+                <input type="button" value="html" class="tech-item item-two"/>
+                <input type="button" value="bootstrap" class="tech-item item-three"/>
+                <input type="button" value="Ruby" class="tech-item item-four"/>
+            </div>
```

Comment:
- [ ] Kindly let's have the technologies as list items instead of inputs because they should not be text inputs, this is just a list of technologies used to build that particular project, see the example below 👇 
```
<ul>
    <li>CSS</li>
    <li>HTML</li>
</ul>
```
this is going to make our code more semantic 😉